docs/serving/services/using-queue-extensions.md

```diff
@@ -0,0 +1,80 @@
+# How to use extensions enabled by QPoptions
+
+Once your cluster is setup with extensions enabled by QPOptions, a Service can decide which extensions it wish to use and how to configure such extensions. Activating and configuring extensions is described here.
+
+## Prerequisites for preparing the cluster
```

Comment:
Probably a good idea to move this section down so that the prerequisites are grouped together.

The Prereqs could be in one section, for example: 

```
## Prerequisites

Before you can use extensions enabled by QPoptions, you must:

- Prepare your cluster:
  - Make sure you are using a Queue Proxy image...
  - Make sure that the cluster config-features is set with...
- Meet the prerequisites in [Creating a Service](...)
```